docs/database-connection-string.md

```diff
@@ -1,85 +1,83 @@
+The database path string is **case sensitive**, even on operating systems like Windows that use case insensitive file paths. Attempting to establish multiple connections to the same database using different path casing, like `C:\Databases\TestDatabase` and `c:\databases\testdatabase`, will fail.
 
-#### Windows
+Starcounter identifies database by the provided path, not by its physical location. Attempt to establish multiple connections to the same database using different paths will fail.
 
-It is not possible to delete database files while `scdata` process is running.
+### OpenMode
 
-#### Linux
+The OpenMode option defines how to interpret the database path on connection. It can have the following string values:
 
-It is possible to delete database files while `scdata` is running, which may lead to a trap.
+Value               | Description
+:------------------ | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+`Open`              | Opens the database directory from the path specified in the Database option, if it exist. If the directory does not exist, or if there is no Starcounter database located there, an exception is thrown.
+`CreateIfNotExists` | Opens the database directory from the path specified in the Database option, if it exist. If the directory does not exist, or if there is no Starcounter database located there, it will create the necessary directories and database files.
 
-Consider the following scenario:
+### StartMode
 
-- A database is created at `/home/database/TestDatabase`.
-- An `scdata` process is started for the database.
-- The database files are deleted.
-- New database is created in exactly the same location.
-- An app tries to connect to the new database.
-- Starcounter detects, that there is a running `scdata` process associated with this database path.
-- The app connects to the existing `scdata` process, which still operates on the deleted files.
+The StartMode option defines how to start and/or connect to the database process once the database files have been located and/or created.
 
-### The `DatabaseOpenMode` enumeration values
+Value               | Description
+:------------------ | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+`RequireStarted`    | Expects the database process to already be running and will connect to it if it is, or throw an exception if it is not.
+`StartIfNotRunning` | Connects to the database process if it is already running, or starts and connects to it if it is not.
+`StartExclusive`    | Expects no database process to be already running, and starts and connects to a new process if that is the case. If a database process is already running, an exception is thrown.
```

Comment:
```suggestion
`StartExclusive`    | Expects no database process to be already running, starts and connects to a new process if that is the case. If a database process is already running, an exception is thrown.
```

Reply:
This is a tricky use of punctuation. Your edit actually makes the sentence mean something different, since the "and" you're removing is not used to end an enumeration but for joining two propositions. Regardless, I think I'll just rewrite it so it's not so weird.